nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -391,6 +397,73 @@ TfOpNodes buildOpNodes(const GraphRepresentation &representation,
       tf_node->weightReorder(node_count);
     }
 
+    if (tf_node->getOpType() ==
```

Comment:
Not this pr, but later, it might be better to separate these graph traversals as compiler optimizers.

Reply:
I agree with you, Thank you for Review